api/validator.go

```diff
@@ -10,24 +10,50 @@ func validateRegisterV1(c *gin.Context) {
 	rules := getBoletoFromContext(c).Title.Rules
 	bn := getBankFromContext(c).GetBankNumber()
 
+	errorResponse := models.BoletoResponse{
+		Errors: models.NewErrors(),
+	}
+
 	if rules != nil {
-		c.AbortWithStatusJSON(400, models.NewSingleErrorCollection("MP400", "title.rules not available in this version"))
+		errorResponse.Errors.Append("MP400", "title.rules not available in this version")
+		c.AbortWithStatusJSON(400, errorResponse)
 		return
 	}
 
 	if bn == models.Stone {
-		c.AbortWithStatusJSON(400, models.NewSingleErrorCollection("MP400", "bank Stone not available in this version"))
+		errorResponse.Errors.Append("MP400", "bank Stone not available in this version")
+		c.AbortWithStatusJSON(400, errorResponse)
 		return
 	}
 }
 
 //validateRegisterV2 Middleware de validação das requisições de registro de boleto na rota V2
 func validateRegisterV2(c *gin.Context) {
 	r := getBoletoFromContext(c).Title.Rules
+	f := getBoletoFromContext(c).Title.Fees
```

Comment:
Melhor usar o _**title**_ e seus métodos de validação.
```suggestion
	t := getBoletoFromContext(c).Title
```

Reply:
done